x-pack/plugins/security_solution/public/flyout/left/components/prevalence_details.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+import { getSummaryRows } from '../../../common/components/event_details/get_alert_summary_rows';
+import { LeftPanelContext } from '../context';
+import { PrevalenceDetails } from './prevalence_details';
+import {
+  PREVALENCE_DETAILS_TABLE_ERROR_TEST_ID,
+  PREVALENCE_DETAILS_TABLE_TEST_ID,
+} from './test_ids';
+import { useFetchFieldValuePairByEventType } from '../../shared/hooks/use_fetch_field_value_pair_by_event_type';
+import { useFetchFieldValuePairWithAggregation } from '../../shared/hooks/use_fetch_field_value_pair_with_aggregation';
+import { useFetchUniqueByField } from '../../shared/hooks/use_fetch_unique_by_field';
+
+jest.mock('../../../common/components/event_details/get_alert_summary_rows');
+jest.mock('../../shared/hooks/use_fetch_field_value_pair_by_event_type');
+jest.mock('../../shared/hooks/use_fetch_field_value_pair_with_aggregation');
+jest.mock('../../shared/hooks/use_fetch_unique_by_field');
+
+const panelContextValue = {
+  eventId: 'event id',
+  indexName: 'indexName',
+  browserFields: {},
+  dataFormattedForFieldBrowser: [],
+} as unknown as LeftPanelContext;
+
+describe('PrevalenceDetails', () => {
+  (useFetchFieldValuePairByEventType as jest.Mock).mockReturnValue({
```

Comment:
jest.mocked(useFetchFieldValue) would work better nowadays

Reply:
I've converted most of them to `jest.mocked` but not all of them as we loose the fact that `as jest.Mock` ignores type check. In most scenarios it's good to have type check, but when we care only about a couple of properties on a giant object, using `as jest.Mock` is very convenient :)